specification/schemas/returns/MostReturnedItem.json

```diff
@@ -0,0 +1,53 @@
+{
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "item": {
+      "type": "object",
+      "additionalProperties": false,
+      "required": [
+        "sku",
+        "name",
+        "amount",
+        "return_reasons"
+      ],
+      "properties": {
+        "sku": {
+          "type": "string",
+          "example": "ABC-123"
+        },
+        "name": {
+          "type": "string"
+        },
+        "amount": {
+          "type": "number"
+        },
+        "return_reasons": {
+          "type": "array",
+          "required": [
+            "code",
+            "name",
+            "amount"
```

Comment:
Somehow these are not taken into account. Perhaps then need to go under "items"?
![image](https://github.com/MyParcelCOM/api-specification/assets/46448148/6cb7687c-52f0-48d5-8552-659c8188e1da)